### ISSUE TITLE
Not able to register a new user

### ISSUE THREAD

**Issue by Taras-Zavorotnii** [simpulton/noterious#8]
When running the app on a local box it is not possible to create a new user due to the changes in the Firebox Api.
![2016-08-27 4](https://cloud.githubusercontent.com/assets/21278984/18027957/a9834aa4-6c71-11e6-97a1-f72e6858a20a.png)

**Comment by katowulf** [comment]
Correct. This tutorial was written over a year ago and Firebase underwent major revisions, including a new SDK and console since. See the migration guide [here](https://firebase.google.com/support/guides/firebase-web) for steps to convert the code to the new SDK.

**Comment by Taras-Zavorotnii** [comment]
I've created a pull request with the migration
